### PR TITLE
KAS-4604: Responsive navigation patterns

### PR DESCRIPTION
**Pull request by Dietr**
First PR to make parts of the Kaleidos app responsive. The current focus is on the interface and navigation patterns. Content sections and content components are for the next PR.

## Scope:
- m-header
- pagination
- subheaders
- sidebar and sidenav
- global spacing

## Affected templates and sections:
- search
- agenda
- cases
- document viewer
- headers and navigation of newsletter/publications/signatures/settings: just to keep navigation consistent, not a focus point of this PR

## Stylesheets changes:
- small update to the order of AU/AUK base/components/utilities so we don't load duplicate bases and respect the ITCSS order of appuniversum
- added local AUK component overrides and some custom utilities to make the interface responsive

## New/updated components:
- Auk::Tabs: added responsive, responsiveBreakpoint and vertical properties
- Auk::TabExpanded: variation to accomodate complex tab content
- Auk::MobileFilters: popout filter wrapper

### CODE REVIEW

File: app/controllers/search.js
Comment (brenner-company) [inline app/controllers/search.js:1]: I've added some functionality here (and within `cases/index`) to make sure the content behind is not scrollable/draggable when the filters are opened (within the mobile view).

That makes 3 instances of the resize event listener (and breakpoint conditional) in total (within this PR). I'll try to implement something more centralized (the next sprint), but no need for this PR to wait for that.

I came across https://github.com/freshbooks/ember-responsive, but the project seems out of date with lots of deprecated parts.

**Comment by brenner-company** [comment]
I think I've covered everything within this PR, but there are still various errors coming up within the Cypress tests. I'll give you an okay when everything checks out (hopefully the coming monday).

File: app/components/agenda/side-nav/agenda.hbs [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <Auk::TabExpanded
   data-test-agenda-side-nav-agenda
   @route="agenda.agendaitems"
-  @current-when="agenda.agendaitems"
+  @current-when={{(eq @currentAgenda.id @agenda.id)}} {{!-- as by LinkTo specs, @current-when can also be a boolean, which is needed here --}}
```
Comment (brenner-company) [inline app/components/agenda/side-nav/agenda.hbs:4]: Reverted this to how it's being done on `dev` as `agenda.agendaitems` doesn't cover all bases.

File: app/controllers/search.js
Comment (brenner-company) [reply app/controllers/search.js:1]: FYI: I've got a story within the current sprint for creating the service/helper to do this.

**Comment by brenner-company** [comment]
Seems like the last change will fix the remaining issue breaking the tests 🤞 If these come out fine, can I merge? Or do you want to give at a final look?

**Comment by Dietr** [comment]
> Seems like the last change will fix the remaining issue breaking the tests 🤞 If these come out fine, can I merge? Or do you want to give at a final look?

Did a final check. 
Made two small tweaks: dropdown had an overflow issue on some vertical tabs and the anchor nav didn't wrap properly on mobile.

If the tests are ok you can merge 🥳

**Review by brenner-company — APPROVED** [review]

**Pull request merged**